src/ravindra_V/Number.java

```diff
@@ -0,0 +1,48 @@
+//Assignment6
+
+package ravindra_V;
+
+class Number {void displayEven(int no) {
+	System.out.println("Even numbers are : ");
+	for (int i=no; i<=15; i=i+2) {
+	System.out.println(i);
+	}
+}
+	
+	void displayDiv1(int no) {
+	System.out.println("Numbers divisble by 5 are :");
+	for (int i = no; no <= 30; no = no+1) {
```

Comment:
no = no + 1 is equivalent to no++
please try to follow best-practice.

Refer below code:
https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/blob/main/src/bhushan/numberGame2.java